api-client/src/main/java/com/takipi/api/client/request/team/ChangeTeamMembersRolesRequest.java

```diff
@@ -0,0 +1,103 @@
+package com.takipi.api.client.request.team;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+import com.takipi.api.client.data.team.ServiceUsers;
+import com.takipi.api.client.data.team.ServiceUsers.TeamMember;
+import com.takipi.api.client.data.team.ServiceUsersResponseMessage;
+import com.takipi.api.client.request.ServiceRequest;
+import com.takipi.api.core.request.intf.ApiPostRequest;
+
+import java.util.List;
+
+public class ChangeTeamMembersRolesRequest extends ServiceRequest implements ApiPostRequest<ServiceUsersResponseMessage>
+{
+	private final ServiceUsers serviceUsers;
+	
+	protected ChangeTeamMembersRolesRequest(String serviceId, ServiceUsers serviceUsers) {
+		super(serviceId);
+		
+		this.serviceUsers = serviceUsers;
+	}
+	
+	@Override
+	public String postData() {
+		return ((new Gson()).toJson(serviceUsers));
+	}
+	
+	@Override
+	public Class<ServiceUsersResponseMessage> resultClass() {
+		return ServiceUsersResponseMessage.class;
+	}
+	
+	@Override
+	public String urlPath() {
+		return baseUrlPath() + "/team/change-role";
+	}
+	
+	public static Builder newBuilder() {
+		return new Builder();
+	}
+	
+	public static class Builder extends ServiceRequest.Builder {
+		private List<TeamMember> members = Lists.newArrayList();

```

Comment:
Unless we allow a user to have multiple roles, this should be map.
Also, please initialize members in the c-tor